diff --git a/video/out/vulkan/utils.c b/video/out/vulkan/utils.c
index d01fc3be0b..d8b695fb28 100644
--- a/video/out/vulkan/utils.c
+++ b/video/out/vulkan/utils.c
@@ -12,12 +12,18 @@ bool mpvk_init(struct mpvk_ctx *vk, struct ra_ctx *ctx, const char *surface_ext)
         surface_ext,
     };
 
+    const char *opt_layers[] = {
+        "VK_LAYER_LS_frame_generation",
+    };
+
     mppl_log_set_probing(vk->pllog, true);
     vk->vkinst = pl_vk_inst_create(vk->pllog, &(struct pl_vk_inst_params) {
         .debug = ctx->opts.debug,
         .debug_extra = ctx->opts.debug,
         .extensions = exts,
         .num_extensions = MP_ARRAY_SIZE(exts),
+        .opt_layers = opt_layers,
+        .num_opt_layers = MP_ARRAY_SIZE(opt_layers),
     });
     mppl_log_set_probing(vk->pllog, false);
     if (!vk->vkinst)
